### PR TITLE
Issue 2080 log screen size

### PR DESCRIPTION
**Pull request by shane-tomlinson**

### CODE REVIEW

**Comment by shane-tomlinson** [comment]
@kparlante - here is the PR we discussed yesterday. I did a bit of additional cleanup, and added some docs.

**Comment by shane-tomlinson** [comment]
fixes #2080

**Comment by shane-tomlinson** [comment]
This breaks on older version of Firefox (18.0.2 tested), but only when run with `?coverage`:

> window._$blanket is undefined
@http://127.0.0.1:3030/tests/index.html?coverage:53
@http://127.0.0.1:3030/tests/spec/lib/screen-info.js:33
callFn@http://127.0.0.1:3030/bower_components/mocha/mocha.js:4338
Runnable.prototype.run@http://127.0.0.1:3030/bower_components/mocha/mocha.js:4331
Runner.prototype.runTest@http://127.0.0.1:3030/bower_components/mocha/mocha.js:4728
Runner.prototype.runTests/next/<@http://127.0.0.1:3030/bower_components/mocha/mocha.js:4806
next@http://127.0.0.1:3030/bower_components/mocha/mocha.js:4653
Runner.prototype.hooks/next/<@http://127.0.0.1:3030/bower_components/mocha/mocha.js:4663
next@http://127.0.0.1:3030/bower_components/mocha/mocha.js:4601
Runner.prototype.hook/<@http://127.0.0.1:3030/bower_components/mocha/mocha.js:4630
timeslice@http://127.0.0.1:3030/bower_components/mocha/mocha.js:5761

**Comment by shane-tomlinson** [comment]
@zaach, @vladikoff and @kparlante - can you check this out?

File: app/scripts/lib/app-start.js [hunk 1]
```diff
@@ -192,13 +194,19 @@ function (
 
     initializeMetrics: function () {
       var relier = this._relier;
+      var screenInfo = new ScreenInfo(window);
```
Comment (vladikoff) [inline app/scripts/lib/app-start.js]: Should this use  `this._window` of app-start? 
```js
function Start(options) {
    options = options || {};

    this._window = options.window || window;
    ...
```

Comment (shane-tomlinson) [inline app/scripts/lib/app-start.js]: @vladikoff - yes, yes it should. Great catch.

**Comment by zaach** [comment]
LGTM

**Pull request merged**